protostar/starknet/cheaters/contracts.py

```diff
@@ -288,3 +297,29 @@ async def call(
             general_config=StarknetGeneralConfig(),
         )
         return result.retdata
+
+    async def expect_call(
```

Comment:
I would put it in `ExpectCheaters`, contract cheaters are for legal starknet operations :police:

Reply:
[add 'expects' cheater, modify cheaters hierarchy](https://github.com/software-mansion/protostar/pull/1363/commits/650efd7813074d4654e979c9a21a14da3dffad95)